src/Writers/Writers.jl

```diff
@@ -0,0 +1,51 @@
+"""
+Provides a rendering function, [`render`](@ref), for writing each supported
+[`Formats.Format`](@ref) to file.
+
+Note that currently `Formats.Markdown` is the **only** supported format.
+
+# Internals
+
+The code for each format should generally be in a separate file (e.g. `markdown.jl`)
+and wrapped in a module. In the interest of code readability, the module names
+should be suffixed with a "W", to indicate that it is a "Writer" (e.g. `MarkdownW`),
+even if there is not direct conflict.
+
+"""
+module Writers
+
+import ..Documenter:
+
```

Comment:
Fair enough, though all the other `import`s should be fixed as well at some point. Preferably in a separate PR later on.